components/ui/stickerSelector.tsx

```diff
@@ -0,0 +1,41 @@
+import { useStickerStore } from "@/stores/sticker.store";
+import { useTranslations } from "next-intl";
+import Image from "next/image";
+import { stickerItems } from "./stickerItems";
+
+type Sticker = {
+  id: number;
+  path: string;
+  bgPath: string;
+};
+
+const StickerSelector = () => {
+  const t = useTranslations("StickerSelector");
+
+  const stickerId = useStickerStore(
+    (state: { stickerId: number }) => state.stickerId,
+  );
+  const updateSticker = useStickerStore(
+    (state: { updateSticker }) => state.updateSticker,
+  );
+  return (
+    <div className="z-2 absolute mt-[154px] flex h-[560px] w-[342px] flex-col items-center gap-6 rounded-xl bg-[#F8F8F7] py-6">
+      <p className="text-base font-semibold text-[#6C1F1F]">{t("sticker")}</p>
+      <div className="flex w-[253px] flex-wrap items-center gap-x-[50px] gap-y-4">
+        {stickerItems.map((item: Sticker) => (
+          <div
+            key={item.id}
+            className={
+              stickerId === item.id ? "rounded-lg border border-[#AA5656]" : ""
+            }
```

Comment:
better refactor

```ts
import { cn } from "@/lib/utils";

className={cn(stickerId === item.id && "rounded-lg border border-[#AA5656]" )}
```

More information on using `cn()` which is a wrapper of `clsx()`: https://github.com/lukeed/clsx